docs/conf.py

```diff
@@ -0,0 +1,217 @@
+    r"http://127.0.0.1",
+    r"https://www.linode.com/",
+    r"https://github.com/plone/documentation/issues/new/choose",  # requires auth
+]
+linkcheck_anchors = False
+linkcheck_timeout = 10
+
+# This is our wordlist with known words, like Github or Plone ...
+spelling_word_list_filename = "spelling_wordlist.txt"
+spelling_ignore_pypi_package_names = True
+
+# The suffix of source filenames.
+source_suffix = {
+    ".md": "markdown",
+}
+
+# The master toctree document.
+master_doc = "index"
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "spelling_wordlist.txt",
+    "**/CHANGES.rst",
+    "**/LICENSE.rst",
+]
+
+
+# -- Options for myST markdown conversion to html -----------------------------
+
+# For more information see:
+# https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
+myst_enable_extensions = [
+    "deflist",  # You will be able to utilise definition lists
+                # https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#definition-lists
+    "linkify",  # Identify “bare” web URLs and add hyperlinks.
+    "colon_fence",  # You can also use ::: delimiters to denote code fences,\
+                    #  instead of ```.
+]
+
+
+# -- Intersphinx configuration ----------------------------------
+
+# This extension can generate automatic links to the documentation of objects
+# in other projects. Usage is simple: whenever Sphinx encounters a
+# cross-reference that has no matching target in the current documentation set,
+# it looks for targets in the documentation sets configured in
+# intersphinx_mapping. A reference like :py:class:`zipfile.ZipFile` can then
+# linkto the Python documentation for the ZipFile class, without you having to
+# specify where it is located exactly.
+#
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
+#
+intersphinx_mapping = {
+    "training": ("https://training.plone.org/5/", None),
+    "python": ("https://docs.python.org/3/", None),
+}
+
+
+# -- GraphViz configuration ----------------------------------
+
+graphviz_output_format = "svg"
+
+
+# -- OpenGraph configuration ----------------------------------
+
+ogp_site_url = "https://training.plone.org/5/"
+ogp_description_length = 200
+ogp_image = "https://docs.plone.org/_static/Plone_logo_square.png"
+ogp_site_name = "Plone Documentation"
+ogp_type = "website"
+ogp_custom_meta_tags = [
+    '<meta property="og:locale" content="en_US" />',
+]
+
+
+# -- sphinx_copybutton -----------------------
+copybutton_prompt_text = r"^ {0,2}\d{1,3}"
+copybutton_prompt_is_regexp = True
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_book_theme"
+
```

Comment:
custom.css and print.css are not included in _build. I've build with `make html` and `gulp`.
With explicit html_static_path they are included.

```suggestion

html_static_path = ["_static"]

```

Reply:
Ah, yes, thank you! Fixed in de0b83a5b234f7909c6c2d9883003dcad0d1055b